kairos-cli/bin/error.rs

```diff
@@ -0,0 +1,15 @@
+use crate::crypto::error::CryptoError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum CliError {
+    /// Unable to find argument by name.
+    #[error("missing argument '{context}'")]
+    MissingArgument { context: &'static str },
+    /// Failed to parse amount from string.
+    #[error("failed to parse '{context}' as u64")]
+    FailedToParseU64 { context: &'static str },
```

Comment:
these are errors that clap has built in

Reply:
Error handling improved in https://github.com/cspr-rad/kairos/pull/17/commits/f54366613e48e8b5f9f5410a26b2b43124965881, now those error are handled by `clap`.